spec/controllers/users_controller_spec.rb

```diff
@@ -54,4 +55,40 @@
       it { expect(response).to redirect_to(user_account_index_exchanges_hbx_profiles_url) }
     end
   end
+
+  describe '.edit' do
+    let(:user) { FactoryGirl.create(:user, :with_consumer_role) }
+    before do
+      sign_in(admin)
+      get :edit, id: user.id, format: 'js'
+    end
+    it { expect(assigns(:user)).to eq(user) }
+    it { expect(response).to render_template('edit') }
+  end
+
+  describe '.update' do
+    let(:user) { FactoryGirl.create(:user, :with_consumer_role) }
```

Comment:
per comment on the update.js.erb

If you modify the controller to check if user.errors.empty? or present? - we can then mock in the controller without needing to actually set any values.

ex)
`allow(user).to receive(:update_attributes).with(email).and_return(true)`
`allow(user).to receive(:errors).and_return({email: 'Email is already taken'})`

may not be exactly correct